pcsx2/MemoryCardFolder.cpp

```diff
@@ -488,7 +488,7 @@ bool FolderMemoryCard::AddFolder(MemoryCardFileEntry* const dirEntry, const std:
 			if (file.m_isFile)
 			{
 				// don't load files in the root dir if we're filtering; no official software stores files there
-				if (enableFiltering && parent == nullptr)
```

Comment:
I understand _how_ this works (we shouldn't care about filtering or not, loose files should not exist in the folder's root, if they do they are made by homebrew to which I say too bad, do what games do and make a folder). I would _infer_ the original purpose behind the filtering check was that homebrew could run with filtering off and then have loose files in the root, but per my previous sentence I think that's just too bad and homebrew should update to match the standards retail games used, instead of having more jank littered in folder memcard code to make them work.

Conversely, an argument could be made that while Sony's standards were that loose files should not exist on the root, the memcards themselves did not have any control logic to prevent this and it was indeed legal at a hardware level. It was only ever prohibited by SCE policing licensed releases. So you could argue that the folder memcard implementation should also mirror that, instead of just hitting a continue keyword because we're in the root.

I think this is fine for now, for the sake of getting this fixed, but certainly down the line I think this should be revisited and perhaps reworked. This comment isn't something to fix, more like a note to myself or whoever else gets to this, about some considerations for what the logic should be.